modules/nf-scil/reconst/fodf/meta.yml

```diff
@@ -1,10 +1,14 @@
 ---
 name: "reconst_fodf"
 description: Perform FODF reconstruction and compute FODF metrics from a dwi volume for a selected number of shells.
+  Note that multiple choices of FODF reconstruction method are available through the method argument.
+  The single-shell single-tissue (ssst) method can be performed by selecting only one shell and choosing ssst as method.
+  The multi-shell single-tissue (msst) method (default) can be performed by selecting multiple shells and choosing ssst as method.
+  The multi-shell multi-tissue (msmt) method can be performed by selecting multiple shells and choosing msmt as method.
```

Comment:
Here :
- add the kind of output expected form using the different methods.
- replace the `can be` by `is`, it's not one way for a method, it's the only one.
- since we don't have meta for `task.ext` give the explicit names to the params to use to select the shells and the method. Something like : `selecting multiple shells (task.ext.fodf_shells) and choosing the ssst method (task.ext.method)`.